docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
     logging: *default-logging
     restart: always
   musearch:
-   image: semtech/mu-search:0.6.3
+   image: semtech/mu-search:0.7.0
```

Comment:
Heb eventjes de mu-search repo bekeken.
Deze comment hoort op zich niet hier thuis, maar aangezien de versie bump hier gebeurt zit ik het er even bij.

In 0.7.0 is die has en has-no erbij gekomen om te gebruiken in de search
Die has en has-no werkt blijkbaar enkel met de waarde 't' ?
Dat strookt een beetje tegen de manier van werken t.o.v. andere delen van de stack (resource)

In een query/call naar resource kan je wel andere values meegeven `':has-no:agenda-activities': 'yes',` 
Ik vermoed dat in die query eender welke waarde resulteert in een true.

Voor de rest is deze PR in orde

Reply:
Terechte opmerking. Resources werkt (gedocumenteerd) enkel met waarde `yes`. Zie discussie op https://github.com/mu-semtech/mu-search/pull/20 . Momenteel is het mu-search-intern consistent. We proberen het later breder uniform te maken.